packages/pluginutils/src/normalizePath.ts

```diff
@@ -2,7 +2,7 @@ import { win32, posix } from 'path';
 
 import { NormalizePath } from '../types';
 
-const normalizePath: NormalizePath = function(filename: string) {
+const normalizePath: NormalizePath = function (filename: string) {
```

Comment:
Was there a change in prettier here? IIRC the previous style is the one used by prettier.

Reply:
yeah, I didn't do this. it was auto-formatted this way